web/template/add-assurance-visit.gotmpl

```diff
@@ -1,10 +1,14 @@
 {{ template "page" . }}
+
+{{ define "page-name" }}Add assurance visit{{ end }}
+
 {{ define "main" }}
-{{ template "content-header" . }}
-{{ template "error-summary" (rename_errors .Errors) }}
+    {{ template "content-header" . }}
+    {{ $RenamedErrors := (rename_errors .Errors) }}
```

Comment:
If this is being used in multiple places now, it is probably better to do it in the Go handler  and just pass in the renamed errors as a variable.